src/common/const.ts

```diff
@@ -77,6 +78,7 @@ export const DOMAINS_WITH_CARD = [
   "input_text",
   "lock",
   "media_player",
+  "number",
```

Comment:
This means you would need to add a specific state card for this domain, like the `input_number` also has:
<https://github.com/home-assistant/frontend/blob/dev/src/state-summary/state-card-input_number.js#L26>